sips/sip-031/sip-031.md

```diff
@@ -0,0 +1,575 @@
+
+> *Other ecosystems that updated their treasury operations and allocated material growth funding have successfully unlocked new products, achieved roadmap milestones, improved liquidity, and enabled real-time, transparent funding for builders-outcomes that are now considered essential for ecosystem leadership.*
+> - **RENDER**: Render replaced $RENDER payments with a burn-and-mint equilibrium (BME) model to boost efficiency and sustainability in their decentralized GPU rendering marketplace. This update involved a planned increase of the total supply by about 20%, to be released in small batches (never exceeding 10% per year). The BME model has been live since December 2023, and within one year, it nearly tripled on-chain network usage compared to 2023\.
+> - **Sui**: Since the announcement of its ecosystem fund worth over $50M to support its DeFi ecosystem in Q4 2023, **Sui** has seen explosive growth, with TVL increasing by \~50x, DEX volumes by \~20x, and daily active users \~15x as of Q1 2025\. The comparison shows clear growth across all key metrics, which could suggest that the initial funding helped drive ecosystem growth.
+> - **Avalanche**: Since the announcement of its $180M DeFi incentive program in Q3 2021, **Avalanche** has grown steadily, with TVL increasing by \~8x, DEX volumes by \~2x, and daily active users by \~6x as of Q1 2025\. The comparison shows clear growth across all key metrics, which could suggest that the initial funding helped drive ecosystem growth.
+> - **NEAR**: Since the announcement of its $800M funding initiative in Q4 2021, **NEAR** has seen significant growth, with TVL increasing by \~4x, fees by \~3x, and daily active users by \~160x as of Q1 2025\. The comparison shows clear growth across all key metrics, which could suggest that the initial funding helped drive ecosystem growth.
+
+
+### **Treasury Impact**
+
+- **Increased development activity:** Consistent funding for core protocol developers to drive development and innovation.  
+- **Ecosystem growth:** Grants and incentives will attract developers, projects, and users to the Stacks ecosystem.  
+- **Enhanced marketing and adoption:** Dedicated resources towards marketing will increase awareness and adoption of Stacks.  
+- **Improved market dynamics:** Active treasury management and more robust capital markets activities can contribute to a healthier STX market.
+
+### **Treasury Allocations & Usage**
+
+**Building for the Long Run: Roadmap and Stacks Value Accrual**
```

Comment:
Is this meant to be a sub-heading?

Reply:
I went to the author's latest branch: https://github.com/stacksgov/sips/blob/cuevasm-patch-1/sips/sip-031/sip-031.md?plain=1

found this
<img src="https://github.com/user-attachments/assets/2dac7c2d-4682-4cac-bc66-398b84e34f11" width=40% height=40%>

Seems to have been changed to sub-heading. 

Resolved?